README.md

```diff
@@ -1,65 +1,130 @@
-# Logseq
+<!-- logo -->
+<p align="center">
+    <a href="https://logseq.com" alt="Logseq Website">
+    <img src="https://raw.githubusercontent.com/logseq/logseq/master/resources/icon.png?sanitize=true" height="173"/></a>
+</p>
+
+<h1 align="center"> Logseq </h1>
+
+<h4 align="center">
+    A privacy-first, open-source platform for knowledge management and collaboration
+</h4>
+
+<div align="center">
+    <a href="https://logseq.com">Home Page</a> |
+    <a href="https://blog.logseq.com/">Blog</a> |
+    <a href="https://docs.logseq.com/">Documentation</a> | 
+    <a href="https://trello.com/b/8txSM12G/roadmap">Roadmap</a>
+</div>
+<br></br>
+
+<p align="center">
+    <a href="https://github.com/logseq/logseq/releases/latest/">
+        <img src="https://img.shields.io/badge/Download_Logseq-100000?style=for-the-badge&logo=flatpak&logoColor=white&labelColor=002b36&color=85c8c8"
+            alt="Download Logseq"/></a>
+</p>
+
+<!-- social badges -->
+<p align="center">
+    <a href="https://discuss.logseq.com">
+        <img src="https://img.shields.io/badge/forum-Logseq-blue.svg?&color=%2385c8c8&logo=discourse&logoColor=%2385c8c8&style=for-the-badge"
+            alt="forum"></a>
+    <a href="https://discord.gg/KpN4eHY">
+        <img src="https://img.shields.io/discord/725182569297215569?color=%2385c8c8&label=Discord&logo=discord&logoColor=%2385c8c8&style=for-the-badge"
+            alt="chat on Discord"></a>
+    <a href="https://twitter.com/intent/follow?screen_name=logseq">
+        <img src="https://img.shields.io/twitter/follow/logseq?color=%2385c8c8&label=%40logseq&logo=twitter&logoColor=%2385c8c8&style=for-the-badge"
+            alt="follow on Twitter"></a>
+</p>
+
+<!-- dev badges -->
+<p align="center">
+    <a href="https://github.com/logseq/logseq/graphs/contributors" alt="Contributors">
+        <img src="https://img.shields.io/github/contributors/logseq/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="#🫶-support-logseq-development" alt="Backers on Open Collective">
+        <img src="https://img.shields.io/opencollective/backers/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="#💎-sponsors" alt="Sponsors on Open Collective">
+        <img src="https://img.shields.io/opencollective/sponsors/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="https://github.com/logseq/logseq/blob/master/LICENSE.md" alt="Activity">
+        <img src="https://img.shields.io/github/license/logseq/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="https://github.com/logseq/logseq/releases">
+        <img src="https://img.shields.io/github/v/release/logseq/logseq?color=%2385c8c8&style=for-the-badge"
+            alt="latest release version"></a>
```

Comment:
I have found an interesting badge that indicates the download count of all releases.  I don't know whether to add it or not. The badge looks convincing.

```markdown
![GitHub all releases](https://img.shields.io/github/downloads/logseq/logseq/total?style=for-the-badge)
```

![GitHub all releases](https://img.shields.io/github/downloads/logseq/logseq/total?style=for-the-badge)

Reply:
I had it there. User feedback suggested that it's not necessary and confusing to average users. It's a cool stat, though!